README.md

```diff
@@ -110,6 +85,7 @@ As a result, `renderedHtml` becomes a string `<h1><a href="http://example.com">E
 
 
 ## Release History
+* 0.3.1 - Make it compatible with Webpack 3.0 and update dependencies
```

Comment:
This will be a breaking change and thus needs to be a major version.

Reply:
True, will change it.